truenas/webui
# NAS-123110 / 23.10 / NFS 'Add SPN' option should be presented after 'Require Kerberos' is in DB

**Testing**

On the **System Settings \> Services \> NFS** page,

To reproduce the issue, follow these steps:

1. Open the NFS configuration page.
2. Select the **Require Kerberos for NFSv4** option.
3. Click **Add SPN** button (adjust one of the conditions on which it's displayed, which checks the system-wide kerberos support)
4. Fill **Add SPN dialog** and submit
5. Save NFS configuration

* Expected Result: First the **Require Kerberos for NFSv4** setting is saved, then the request `nfs.add_principal` is made, but not earlier.
